ui/app/controllers/access-control/policies/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
```

Comment:
In addition to routes (load the resources we need), and components (shared functionality for reusable pieces of code), there are updates to several "controller" files in this PR.

A controller in Ember is tied to a particular route, and helps define functionality for the template that can be found there.

Here, for example, we hydrate the `policies` list with the count of tokens each has, to make for easier sorting by the end-user. We also provide delete functionality with a notification on success/error, etc.